ansible/roles/containerd/tasks/install.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# Check if containerd file already exists
+# used to skip the download when offline
+- name: check if the containerd tar exists
+  changed_when: false
+  stat:
+    path: "{{ containerd_remote_bundle_path }}/{{ containerd_tar_file }}"
+  register: containerd_tar_file_exists
+
+- name: create containerd bundle directory
+  file:
+    path: "{{ containerd_remote_bundle_path }}"
+    state: directory
+  when:
+    - not containerd_tar_file_exists.stat.exists  
+
+- name: download containerd
+  get_url:
+    url: "{{ containerd_url }}"
+    dest: "{{ containerd_remote_bundle_path }}/{{ containerd_tar_file }}"
+    mode: 0600
+  when:
+    - not containerd_tar_file_exists.stat.exists  
+
+- name: unpack containerd
+  unarchive:
+    remote_src: True
+    src: "{{ containerd_remote_bundle_path }}/{{ containerd_tar_file }}"
+    dest: /
+    extra_opts:
+      - --no-overwrite-dir
+
+# Remove /opt/cni directory, as we will install cni later
+- name: delete /opt/cni directory
+  file:
+    path: /opt/cni
+    state: absent
+
+# Remove /etc/cni directory, as we will configure cni later
+- name: delete /etc/cni directory
+  file:
+    path: /etc/cni
+    state: absent
```

Comment:
Our containerd tarballs will not include CNI. We can remove these workarounds in the future.

Reply:
Removed